include/triton/core/tritonbackend.h

```diff
@@ -94,7 +94,7 @@ struct TRITONBACKEND_Batcher;
 ///   }
 ///
 #define TRITONBACKEND_API_VERSION_MAJOR 1
-#define TRITONBACKEND_API_VERSION_MINOR 13
+#define TRITONBACKEND_API_VERSION_MINOR 14
```

Comment:
This will have to be 15 because of my PR: https://github.com/triton-inference-server/core/pull/235